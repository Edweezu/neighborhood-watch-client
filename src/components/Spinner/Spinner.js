import React from 'react'
// import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
// import { faBowlingBall } from '@fortawesome/free-solid-svg-icons'
class Spinner extends React.Component {

  componentDidMount () {
    document.body.style.overflowY = 'hidden'
  }

  componentWillUnmount () {
    document.body.style.overflowY = 'auto'
  }

  render () {
    return (
      <div className='rolling'>
      <div className='spinner'>
      {/* <FontAwesomeIcon icon={faBowlingBall} size='3x' color='#3B5998' /> */}
        <i className="fas fa-bowling-ball spinner" size='5x' color='#1D3C4C'></i>
      </div>
    </div>
    
    )
  }
}

export default Spinner
 