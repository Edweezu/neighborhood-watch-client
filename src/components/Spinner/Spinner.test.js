import React from 'react'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'
import Spinner from './Spinner'


describe('Spinner component', () => {
    it('renders the given component', () => {
        const wrapper = shallow(<Spinner />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})