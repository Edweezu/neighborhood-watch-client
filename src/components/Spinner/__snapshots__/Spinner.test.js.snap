// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component renders the given component 1`] = `
<div
  className="rolling"
>
  <div
    className="spinner"
  >
    <i
      className="fas fa-bowling-ball spinner"
      color="#1D3C4C"
      size="5x"
    />
  </div>
</div>
`;
